app/routes.py

```diff
@@ -376,3 +376,32 @@ def remove_candidate():
 @app.route("/404", methods=["GET"])
 def missing_route():
     return render_template("404.html")
+
+
+# the update status function is the function to be ran repaeatedly hence the id
+# the function will be ran in 30 seconds intervals
+@scheduler.task('interval', id='do_update_status', seconds=10, misfire_grace_time=900)
```

Comment:
Nice work. But we don't need this function to keep on running every 30 secs for all elections. There could be huge performance drawbacks. Why can't we set a schedule event to fire on the time the owner set. As per can't the scheduler ring at a set time? If so then can't we set it to execute at the ending_at?